sites/fast-website/src/app/css/style.css

```diff
@@ -60,7 +60,8 @@ body {
 }
 
 .header {
-  grid-column: 2;
+  grid-column: 1/4;
```

Comment:
May I suggest using grid-column: 1/-1 in this case?  That way you have full width even if columns are added or removed at some point.

Reply:
yeah, good point.